.github/workflows/Consitency.yaml

```diff
@@ -0,0 +1,36 @@
+name: Consistency
+
+on: [push, pull_request]
+
+jobs:
+  Unit:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up PHP Version ${{ matrix.php }}
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: 7.4
+          tools: composer:v2
+          coverage: none
+
+      - name: Environment Check
+        run: |
+          php --version
+          composer --version
+
+      - name: "Determine tag"
+        id: "determine-tag"
+        run: "echo \"::set-output name=tag::${GITHUB_REF#refs/tags/}\""
+
+      - name: Consitency Checks
```

Comment:
I've noticed a small typo here: consitency -> **consistency**. The same typo is in the filename as well.

Reply:
Thanks, fixed now